packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -271,10 +284,18 @@ export class ClustersService extends ImmutableStore<ClustersServiceState> {
   }
 
   async removeGateway(gatewayUri: string) {
-    await this.client.removeGateway(gatewayUri);
-    this.setState(draft => {
-      draft.gateways.delete(gatewayUri);
-    });
+    try {
+      await this.client.removeGateway(gatewayUri);
+      this.setState(draft => {
+        draft.gateways.delete(gatewayUri);
+      });
+    } catch (error) {
+      this.notificationsService.notifyError({
+        title: 'Could not close DB connection',
+        description: error.message,
+      });
+      throw error;
```

Comment:
Same question as for cluster resources, how can I trigger this path when using the app?